src/components/buttons/styled.ts

```diff
@@ -0,0 +1,66 @@
+import { Colour, theme } from '@fellesdatakatalog/theme';
+import styled from 'styled-components';
+
+const BigButton = styled.button<{ $inverse?: boolean }>`
+  border: 1px solid;
+  border-radius: 2px;
+  border-color: ${({ $inverse }) =>
+    $inverse ? ({ theme: t }) => t.entityColours.dark : 'transparent'};
+
+  padding: ${theme.spacing('S8')} ${theme.spacing('S24')};
+  background-color: ${({ $inverse }) =>
+    $inverse
+      ? theme.colour(Colour.NEUTRAL, 'N0')
+      : ({ theme: t }) => t.entityColours.dark};
+
+  color: ${({ $inverse }) =>
+    $inverse
+      ? ({ theme: t }) => t.entityColours.dark
+      : theme.colour(Colour.NEUTRAL, 'N0')};
+
+  & path {
+    stroke: ${({ $inverse }) =>
+      $inverse
+        ? ({ theme: t }) => t.entityColours.dark
+        : theme.colour(Colour.NEUTRAL, 'N0')};
+  }
+
+  &:hover {
+    color: ${({ $inverse }) =>
+      $inverse
+        ? theme.colour(Colour.NEUTRAL, 'N0')
+        : ({ theme: t }) => t.entityColours.dark};
+    background-color: ${({ $inverse }) =>
+      $inverse
+        ? ({ theme: t }) => t.entityColours.dark
+        : ({ theme: t }) => t.entityColours.light};
+
+    & path {
+      stroke: ${({ $inverse }) =>
+        $inverse
+          ? theme.colour(Colour.NEUTRAL, 'N0')
+          : ({ theme: t }) => t.entityColours.dark};
+    }
+  }
+`;
+
+const GhostButton = styled.button`
```

Comment:
Navnet er kanskje misvisende, er det en skjult knapp?

Se evt. om noen av variantene vi har i fdk-kit fra før kan brukes, https://fellesdatakatalog.github.io/fdk-kit/?path=/docs/components-button--tertiary-button ?

Reply:
Nei, GhostButton referer til en knapp uten fyll som her: https://uxplanet.org/ghost-buttons-in-ux-design-4cf3717334f8